src/cryptography/noscrypt_cipher.cpp

```diff
@@ -148,36 +140,40 @@ string NoscryptCipher::naiveEncodeBase64(const std::string& str)
 {
     // Compute base64 size and allocate a string buffer of that size.
     const size_t encodedSize = NoscryptCipher::base64EncodedSize(str.size());
-    unsigned char* encodedData = new unsigned char[encodedSize];
+    
+    unique_ptr<uint8_t> encodedData = make_unique<uint8_t>(encodedSize);
```

Comment:
Minor detail, but if the assignment already has type information, we can use the `auto` keyword to reduce redundancy in the declaration.

```suggestion
    auto encodedData = make_unique<uint8_t>(encodedSize);
```